api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -540,6 +540,35 @@ private boolean userHasEverLoggedIn(
     return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
   }
 
+  @AuthorityRequired(Authority.ACCESS_CONTROL_ADMIN)
+  @Override
+  public ResponseEntity<Profile> adminUpdateProfile(Profile updatedProfile) {
+    final String username = updatedProfile.getUsername();
+    final DbUser user = userService.getByUsernameOrThrow(username);
+
+    // Save current profile for audit trail.
+    final Profile previousProfile = profileService.getProfile(user);
+
+    profileService.validateUpdatedProfile(updatedProfile, previousProfile);
+
+    profileService.adminUpdateProfile(updatedProfile);
+
+    // update the user's free-tier-active status, if this changes
+    if (!updatedProfile.getFreeTierDollarQuota().equals(previousProfile.getFreeTierDollarQuota())) {
+      freeTierBillingService.setFreeTierDollarOverride(
+          user, updatedProfile.getFreeTierDollarQuota());
+    }
+
+    if (workbenchConfigProvider.get().featureFlags.requireInstitutionalVerification) {
+      profileService.saveVerifiedInstitutionalAffiliation(user, updatedProfile);
+    }
+
+    final Profile appliedUpdatedProfile = profileService.getProfile(user);
+    profileAuditor.fireUpdateAction(previousProfile, appliedUpdatedProfile);
```

Comment:
⚠️ You need to update the `ProfileTargetProperty` enum so that the audit log will pick up changes to institutional affiliation (assuming you don't want to make Institution its own target type and CHANGE_AFFILIATION_TO and CHANGE_AFFILIATION_FROM into action types). I would've probably done it the hard way, but we should be able to query for these either way. 

The key point to consider is whether consumers of the log should consider all these changes "profile changes" or more serious admin changes.